pages/services/beta-dcos-monitoring/0.4.3-beta/operations/alertmanager/storage/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: layout.pug
+navigationTitle: Alert Manager Storage
+title: Alert Manager Storage
+menuWeight: 20
+excerpt: Alert Manager Storage
+render: mustache
+model: ../../../data.yml
+---
+
+#include /services/include/beta-software-warning.tmpl
+
+# Configuring Alert Manager Storage
+
+## DSS Volume
+
+You can configure Alert Manager to use a dedicated volume provided by the [DC/OS Storage Service](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/) (DSS).
+
+At least one volume must be available for Alert Manager to use. For more on creating volumes, see the [DSS documentation](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/cli-references/dcos-storage-volume/dcos-storage-volume-create).
```

Comment:
Is there any minimal requirements for this volume that we should recommend?  Should Alert Manager volumes be on high performance storage (e.g. SSD backed)?  Is the example size below (512MB) sufficient?  Or should it be larger.  I would definitely include a recommendation for storage guidelines.

Reply:
I used the default size values from dcos-monitoring's configuration. I've deployed dcos-monitoring a bunch with these storage sizes, so it should be OK.

Reply:
I don't know whether Alert Manager specifically needs fast storage. I think explaining how to decide whether it needs a certain type of storage is out of scope of this doc.

Reply:
If it doesn't need fast storage then a recommendation isn't needed.